corehq/apps/accounting/templates/accounting/partials/confirm_plan_summary.html

```diff
@@ -57,7 +57,16 @@ <h4>
     </h4>
   {% endif %}
 
-  {% if is_upgrade %}
+  {% if is_annual_plan %}
+    <p class="text-free">
+      {% blocktrans %}
+        By continuing you are subscribing to a Pay Annually plan and will receive an invoice via email for plan fees due.
+        In case you incur any on-demand fees for excess users or SMS, you will receive invoices for these charges on a monthly basis.
+        See <a href="https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143955299/Subscription+Management+Billing#Pay-Annually-Subscription">here</a>
+        for more information about payment, cancellation, and refunds.
+      {% endblocktrans %}
+    </p>
+  {% elif is_upgrade %}
     <p class="text-free">
```

Comment:
what is `text-free` class doing it? can we rename it to make it clear it is accounting-specific and should be removed/replaced with bootstrap utils if possible?

Reply:
It's a styling class that's left-aligning the central body text on this alert and giving it a particular width. Renamed here: https://github.com/dimagi/commcare-hq/pull/36448/commits/e89d41bbccd8a9dc901635e1d93176578f6abfdd